netspeed/src/backend.c

```diff
@@ -149,35 +149,30 @@ free_devices_list(GList *list)
 /* Frees a DevInfo struct and all the stuff it contains
  */
 void
-free_device_info(DevInfo *devinfo)
+free_device_info (DevInfo *devinfo)
 {
-	g_free(devinfo->name);
-	g_free(devinfo->ip);
-	g_free(devinfo->netmask);
-	g_free(devinfo->ptpip);
-	g_free(devinfo->ipv6);
-	g_free(devinfo->essid);
+	g_free (devinfo->name);
+	g_free (devinfo->essid);
 #ifdef HAVE_NL
-	g_free(devinfo->tx_bitrate);
-	g_free(devinfo->rx_bitrate);
-	g_free(devinfo->channel);
+	g_free (devinfo->tx_bitrate);
+	g_free (devinfo->rx_bitrate);
+	g_free (devinfo->channel);
 #endif /* HAVE_NL */
+	g_free (devinfo);
 }
 
-static char*
-format_ipv4(guint32 ip)
+static void
+format_ipv4 (guint32  ip,
+             char    *dest)
 {
-	char *str = g_malloc(INET_ADDRSTRLEN);
-	inet_ntop(AF_INET, &ip, str, INET_ADDRSTRLEN);
-	return str;
+    inet_ntop (AF_INET, &ip, dest, INET_ADDRSTRLEN);
 }
 
-static char*
-format_ipv6(const guint8 ip[16])
+static void
+format_ipv6 (const guint8  ip[16],
+             char         *dest)
 {
-	char *str = g_malloc(INET6_ADDRSTRLEN);
-	inet_ntop(AF_INET6, ip, str, INET6_ADDRSTRLEN);
-	return str;
+    inet_ntop (AF_INET6, ip, dest, INET6_ADDRSTRLEN);
```

Comment:
the indentation is wrong

Reply:
No, it isn't.

Reply:
This code block and all other in question has only 1 line.
Does it makes a different if using 4 or 5 spaces?
I do not agree with adding tabs again in newly written code.

Well, maybe you like to discuss indentation issue at https://github.com/mate-desktop/mate-control-center/pull/579 :P